src/main.rs

```diff
@@ -137,7 +137,12 @@ async fn run(do_commit: bool, site_filter: Option<&Regex>) -> anyhow::Result<()>
         .filter(|site| site_filter.map_or(true, |filter| filter.is_match(site.url.as_str())))
         .collect::<Vec<_>>();
     let sites_amount = sites.len();
-    assert!(!sites.is_empty(), "Site filter filtered everything out.");
+    if sites.is_empty() {
+        eprintln!("Error: The Site-filter filtered everything out.
+There are no sites tracked for your query.
```

Comment:
Not sure I like this sentence. Both 'tracked' and 'query' aren't used anywhere else currently. I think the other two sentences already explain what happened so maybe just omit this one?

Reply:
Hm. Fine with me.